packages/@storefront/structure/test/unit/carousel.ts

```diff
@@ -37,12 +37,22 @@ suite('Carousel', ({ expect, spy, stub, itShouldProvideAlias }) => {
     describe('onMount()', () => {
       it('should add event listener for window resize', () => {
         const addEventListener = spy();
+        carousel.set = () => null;
         stub(utils, 'WINDOW').returns({ addEventListener });
 
         carousel.onMount();
 
         expect(addEventListener).to.be.calledWithExactly('resize', carousel.forceUpdate);
       });
+
+      it('should force an update', () => {
+        const set = carousel.set = spy();
```

Comment:
If you replace `set()` with `forceUpdate()`, these tests will have to be updated as well.